presto-docs/src/main/sphinx/security/query-access.json

```diff
@@ -0,0 +1,20 @@
+{
+  "catalogs": [
+    {
+      "allow": true
+    }
+  ],
+  "queries": [
+    {
+      "user": "admin",
+      "allow": ["execute", "kill", "view"]
+    },
+    {
+      "user": "alice",
```

Comment:
I would replace `alice` with `user_than_can_execute_or_kill` same for `allow` user. Also it looks like you are missing a typical case where user is only allowed to `view`.